packages/components/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Internal
 
 -   Replaced `classnames` package with the faster and smaller `clsx` package ([#61138](https://github.com/WordPress/gutenberg/pull/61138)).
+-   Add a vertical toolbar for zoom out mode ([#60123](https://github.com/WordPress/gutenberg/pull/60123))
```

Comment:
```suggestion
-   `Toolbar`: Add support for `vertical` orientation ([#60123](https://github.com/WordPress/gutenberg/pull/60123)).
```

The changelog is for all consumers of the wp-components package, so let's phrase this in a way that's relevant to that audience. Also, I think this should be considered an Enhancement, rather than just Internal.